src/webpack/webpack.ts

```
@@ -83,51 +84,83 @@ export const filters = {
+        // their i18n Proxy "caches" by setting each accessed property to the return, so try to delete
+        Reflect.deleteProperty(value, PROXY_CHECK);
+        return true;
+    }
     if (value instanceof TypedArray) return true;
 
     return false;
 }
 
-export function _shouldIgnoreModule(exports: any) {
-    if (_shouldIgnoreValue(exports)) {
+function makePropertyNonEnumerable(target: Object, key: PropertyKey) {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)